### DISCUSSION TITLE
Add spotlight context

### DISCUSSION THREAD

**Discussion by kurucu** [wire-elements/spotlight#92] [Ideas]
It would be good to be able to add context to spotlight.

For example, if you're viewing a project then you could:

- Add project-specific actions 
- Tailor actions to the project e.g. creating a new customer would be for that project (rather than having to find it again)
- Constrain search within that project

I guess it would be passed in via `@livewire('livewire-ui-spotlight', ['context' => $contextThing])` and then made available to the Commands by spotlight as optional class properties.

**Comment by PhiloNL** [comment]
I've been working on a Pro version of the Spotlight component that supports context behavior. It will be available soon. You can see a little preview here: https://twitter.com/Philo01/status/1556618729860972546
